src/runtime/runtime-test.js

```diff
@@ -447,6 +448,15 @@ describes.realWin('Runtime', {}, env => {
         expect(cr.config().windowOpenMode).to.equal('redirect');
       });
     });
+
+    it('should not return Propensity module when config not available', () => {
+      configPromise = Promise.reject('config not available');
+      return runtime.getExperimentalPropensityModule().then(() => {
```

Comment:
getExperimentalPropensityModulePromise()

Reply:
no need for Promise key word since it could return the module or a promise of that module